app/src/main/java/com/github/houseorganizer/houseorganizer/task/FirestoreTask.java

```diff
@@ -29,16 +33,30 @@ public DocumentReference getTaskDocRef() {
 
     @Override
     public void changeTitle(String newTitle) {
+        EspressoIdlingResource.increment();
         super.changeTitle(newTitle);
 
-        taskDocRef.update("title", newTitle);
+        Task<Void> task = taskDocRef.update("title", newTitle);
```

Comment:
Does this work in the app? I had a Tasks::await call in changeTitle() before, but it made the app crash

Reply:
Good call! It fails for me too. I will delete the await for now then

Reply:
Alright, thanks for the quick update!